src/pages/chaeyeon/Login/Login.js

```diff
@@ -1,5 +1,76 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Login.scss';
+import '../Main/Main';
 
-export default function LoginC() {
-  return <></>;
+function LoginC() {
+  //로그인현재 안보이지만, 아이디 넣고(현재, 아이디 넣은 상태), 비번넣으면(현재,비번넣은상태)
+  const [active, setActive] = useState(false);
+  const [id, setId] = useState('');
+  const [password, setPassword] = useState('');
+  const navigate = useNavigate();
+  const activeLogin = () => {
+    return id.includes('@') && password.length >= 5
+      ? setActive(true)
+      : setActive(false); //비번필수요소
+  };
```

Comment:
- active라는 state는 결국 id, password state값을 조건으로 관리하는 state
- 그렇다면 따로 active state를 만들 필요 없이 id, password state로 계산하면 됨
```suggestion
  const isAllValid = id.includes('@') && password.length >= 5
```

Reply:
closure

Reply:
https://ko.reactjs.org/docs/thinking-in-react.html